src/main.go

```diff
@@ -144,16 +144,7 @@ func newTester(name string) *tester {
 
 func setHashJoinConcurrency(db *sql.DB) {
 	if _, err := db.Exec("SET @@tidb_hash_join_concurrency=1"); err != nil {
-		var ignoreErr = false
```

Comment:
Why ignore error logic is removed?

Reply:
This related to strict mode, and it previously tested if it was a warning here.

Strict mode has been removed from the Go driver: https://github.com/go-sql-driver/mysql/wiki/strict-mode

Reply:
I've merged new commits from master in my branch that fix this issue, so it's now gone from the diff.

@bb7133 PTAL